lib/slack_poster.rb

```diff
@@ -8,6 +8,8 @@ def initialize(webhook_url, team_channel, mood)
     @webhook_url = webhook_url
     @team_channel = team_channel
     @mood = mood
+    @today = Date.today
+    @postable_day = !today.saturday? && !today.sunday?
```

Comment:
That would be a good idea! A "non-work days" attribute could be added to the team config?